buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -12,6 +12,16 @@ object Libs {
    */
   const val appcompat: String = "androidx.appcompat:appcompat:" + Versions.appcompat
 
+  /**
+   * https://developer.android.com/reference/android/support/constraint/ConstraintLayout
+   */
+  const val constraintLayout = "com.android.support.constraint:constraint-layout:" + Versions.constraint_layout
```

Comment:
Let's use the androidx version here as well

Reply:
done